dataland-frontend/src/components/pages/AdminAllRequestsOverview.vue

```diff
@@ -232,15 +235,14 @@ export default defineComponent({
     this.availableFrameworks = retrieveAvailableFrameworks();
     this.availableRequestStatus = retrieveAvailableRequestStatus();
     this.getAllRequestsForFilters().catch((error) => console.error(error));
-    this.resetFilterAndSearchBar();
```

Comment:
Is is deleted because it was not necessary on mount, correct?

Reply:
exactly. on mount the selectionArrays are empty, so no need to reset the filter (which would and did so before, on mount, lead to three api-calls, first due to mount, and two consecutive ones due to resetting the filter).

Reply:
ok